plugins/UsersManager/angularjs/user-edit-form/user-edit-form.component.js

```diff
@@ -142,15 +143,17 @@
                 method: 'UsersManager.updateUser'
             }, {
                 userLogin: vm.user.login,
-                password: vm.isPasswordChanged ? vm.user.password : undefined,
+                password: vm.user.password ? vm.user.password : undefined,
+                passwordConfirmation: vm.passwordConfirmation ? vm.passwordConfirmation : undefined,
                 email: vm.user.email,
                 alias: vm.user.alias
             }).catch(function (e) {
                 vm.isSavingUserInfo = false;
+                vm.passwordConfirmation = false;
                 throw e;
             }).then(function () {
                 vm.isSavingUserInfo = false;
-                vm.isPasswordChanged = false;
+                vm.passwordConfirmation = false;
                 vm.isUserModified = true;
 
                 showUserSavedNotification();
```

Comment:
Haven't tested yet so not sure if this is an issue or not, but should superusers have to confirm the user's password when editing a user? I guess this would only be a problem for them since they're the only ones who can edit other users' info.